src/category_theory/limits/shapes/pullbacks.lean

```diff
@@ -170,16 +200,16 @@ namespace pushout_cocone
 
 variables {f : X ⟶ Y} {g : X ⟶ Z}
 
-def ι₁ (t : pushout_cocone f g) : Y ⟶ t.X := t.ι.app left
-def ι₂ (t : pushout_cocone f g) : Z ⟶ t.X := t.ι.app right
+def inl (t : pushout_cocone f g) : Y ⟶ t.X := t.ι.app left
+def inr (t : pushout_cocone f g) : Z ⟶ t.X := t.ι.app right
 
-def mk {W : C} (ι₁ : Y ⟶ W) (ι₂ : Z ⟶ W) (eq : f ≫ ι₁ = g ≫ ι₂) : pushout_cocone f g :=
+def mk {W : C} (inl : Y ⟶ W) (inr : Z ⟶ W) (eq : f ≫ inl = g ≫ inr) : pushout_cocone f g :=
 { X := W,
   ι :=
-  { app := λ j, walking_span.cases_on j (f ≫ ι₁) ι₁ ι₂,
+  { app := λ j, walking_span.cases_on j (f ≫ inl) inl inr,
     naturality' := λ j j' f, by cases f; obviously } }
 
-lemma condition (t : pushout_cocone f g) : f ≫ (ι₁ t) = g ≫ (ι₂ t) :=
+lemma condition (t : pushout_cocone f g) : f ≫ (inl t) = g ≫ (inr t) :=
```

Comment:
Idem: should this be called `w`. Do we want a `reassoc`?

Reply:
Fixed.